packages/chevrotain/src/parse/parser/parser.ts

```diff
@@ -104,7 +105,7 @@ export interface IParserEmptyAlternativeDefinitionError
 
 export interface IParserAmbiguousAlternativesDefinitionError
   extends IParserDefinitionError {
-  occurrence: number
+  occurrence: number | string
```

Comment:
When is this ever a `string`?

Reply:
https://github.com/Chevrotain/chevrotain/blob/master/packages/chevrotain/src/parse/grammar/checks.ts#L695

But yeah, it would make more sense to have 0 instead of "". I can change it in another PR.

Reply:
Ah I see, may be related to historic code when the identifier of grammar location was built with string concatenation